scripts/run-openapi-validation.ps1

```diff
@@ -16,15 +16,25 @@
 
 .Parameter repoDirectory
     Full path the the root directory of msgraph-metadata checkout.
+
+.Parameter platformName
+    Name of the platform to be tested.
 #>
 
 param(
     [Parameter(Mandatory=$true)][string]$repoDirectory,
-    [Parameter(Mandatory=$true)][string]$version
+    [Parameter(Mandatory=$true)][string]$version,
+    [Parameter(Mandatory=$false)][string]$platformName
 )
-$yaml = Join-Path $repoDirectory "openapi" $version "openapi.yaml"
+
+if([string]::IsNullOrWhiteSpace($aString))
```

Comment:
where is aString supposed to come from?

Reply:
Lemme fix that. It's supposed to be platformName

Reply:
```suggestion
if([string]::IsNullOrWhiteSpace($platformName))
```